src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/mediation/MediationSuccessCallbackHandler.java

```diff
@@ -63,9 +65,16 @@ public MediationSuccessCallbackHandler(
     private CallbackResponse raiseAppInsightEvent(CallbackParams callbackParams) {
         CaseDetails caseDetails = callbackParams.getRequest().getCaseDetails();
         Claim claim = caseDetailsConverter.extractClaim(caseDetails);
-        if (FeaturesUtils.hasMediationPilotFeature(claim)) {
-            appInsights.trackEvent(MEDIATION_PILOT_SUCCESS, REFERENCE_NUMBER, claim.getReferenceNumber());
-        }
+
+        appInsights
+            .trackEvent(getAppInsightEventBasedOnMediationPilot(claim), REFERENCE_NUMBER, claim.getReferenceNumber());
+
         return SubmittedCallbackResponse.builder().build();
     }
+
+    private AppInsightsEvent getAppInsightEventBasedOnMediationPilot(Claim claim) {
+        return FeaturesUtils.hasMediationPilotFeature(claim)
+            ? MEDIATION_PILOT_SUCCESS
+            : NON_MEDIATION_PILOT_SUCCESS;
+    }
```

Comment:
This looks like there is an amount of code duplication between the success and failure callback classes.